### ISSUE TITLE
Clear cache module 2x in admin area 

### ISSUE THREAD

**Issue by djayko** [templaza/astroid-framework#96]
Hi,

I'm not sure, but I've two clear cache options in backend now in status-bar.

And i figured out that there are two modules active and one had a typo.
Only issue on my side?
![grafik](https://user-images.githubusercontent.com/3375044/153765678-53117454-fd35-48b3-8d53-5e47ee73104f.png)

best,
Frank

**Comment by sonvnn** [comment]
Hi Frank,

Please help me remove one of module.

Thanks & Best Regards,
Sonny

**Comment by biobill** [comment]
Hi @djayko,
I can't confirm this issue on my end. I can see only one module without any typo.

**Comment by djayko** [comment]
> Hi @djayko, I can't confirm this issue on my end. I can see only one module without any typo.

Thank you for your feedback.
I tested it again on a second site.

I hat astroid 2.5.13 in place and template zero.
-> I used auto update function of joomla for astroid core and uploaded the newest zero template version (2.5.13) from astroidframe.work

And again two published modules in admin-area.
I'll to 3rd test and maybe i'll get more details.

best,
Frank
